src/runmode-af-xdp.c

```diff
@@ -29,7 +29,6 @@
  * AF_XDP socket runmode
  *
  */
-#define PCAP_DONT_INCLUDE_PCAP_BPF_H 1
```

Comment:
this is checked in pcap.h:
```
grep PCAP_DONT_INCLUDE_PCAP_BPF_H /usr/include/* -R
/usr/include/pcap/pcap.h:#ifndef PCAP_DONT_INCLUDE_PCAP_BPF_H
```

Reply:
It does not hurt, but I do not think we need it here in runmode-af-xdp.c as it does not include <bpf.h>

But we need it in source-af-packet.c where both pcap.h and bpf.h are included